htmd/parameterization/cli.py

```diff
@@ -70,6 +70,35 @@ def getArgumentParser():
 
     return parser
 
+def _prepare_molecule(args):
+
+    from htmd.molecule.molecule import Molecule
+    from htmd.parameterization.util import guessElements
+
+    mol = Molecule(args.filename, guessNE=['bonds'], guess=[])
+
+    # Check if the file contain just one conformation
+    if mol.numFrames != 1:
+        raise RuntimeError('{} has to contain only one molecule, but found {}'.format(args.filename, mol.numFrames))
+
+    # Check if each atom name is unique
+    if np.unique(mol.name).size != mol.numAtoms:
+        raise RuntimeError('The atom names in {} has to be unique!'.format(args.filename))
```

Comment:
It's fine for now, but can be annoying for a user that wants to parameterize a molecule that may have atoms with the same name.

Reply:
This is address with by https://github.com/Acellera/htmd/pull/793